packages/gatsby-source-wordpress/tsconfig.json

```
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "baseUrl": "./src",
+    "paths": {
+      "~/*": ["*"]
+    },
+    "lib": ["ESNext", "DOM"],
+    "target": "ES6",
+    "moduleResolution": "node",
+    "allowSyntheticDefaultImports": true
+  },
+  "include": ["src"]
+}
```

Can we convert to the root tsconfig?
```suggestion
{
  "extends": "../../tsconfig.json"
}
```

---

I think this will break module resolution since we're using a tilde to import from the root. `~/some/path`. I want to remove that anyway because it's not that useful but it will take a bit of time to go through and make sure everything is still importing properly. I think that should be a follow up PR